### PR TITLE
DIG-1471: DateInterval compatibility

### PR DESCRIPTION
**Pull request by mshadbolt**
* Updated the clinical_ingest.json to have date intervals instead of dates
* added date_resolution field
* Updated the clinical_etl_code version to the develop branch

We will need to update the schema url and clinical_etl_code version when those repos are updated

### CODE REVIEW

**Review by SonQBChau — APPROVED** [review]
tested and worked with katsu date intervals

File: requirements.txt [hunk 1]
```diff
@@ -10,4 +10,4 @@ python-dotenv==0.14.0
 dateparser~=1.2.0
 pandas~=2.1.4
 numpy>=1.22.2 # not directly required, pinned by Snyk to avoid a vulnerability
-clinical_etl@git+https://github.com/CanDIG/clinical_ETL_code.git@v1.2.1
+clinical_etl@git+https://github.com/CanDIG/clinical_ETL_code.git@develop
```
Comment (daisieh) [inline requirements.txt]: I forgot to make a release for date intervals, but I just did it [here](https://github.com/CanDIG/clinical_ETL_code/releases/tag/v2.0.0) so this should be v2.0.0.

**Comment by mshadbolt** [comment]
I think we can wait until the Katsu date interval code pr is merged so we can update the schema url in the clinical_ingest.json

**Review by daisieh — APPROVED** [review]
LGTM

**Pull request merged**